modules/nf-core/canu/main.nf

```diff
@@ -0,0 +1,48 @@
+process CANU {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "bioconda::canu=2.2"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/canu:2.2--ha47f30e_0':
+        'quay.io/biocontainers/canu:2.2--ha47f30e_0' }"
+
+    input:
+    tuple val(meta), path(reads)
+    val mode
+    val (genomesize)
+
+    output:
+    tuple val(meta), path("*.report")			, emit: report
+    tuple val(meta), path("*.contigs.fasta")		, emit: assembly		, optional: true
+    tuple val(meta), path("*.unassembled.fasta")	, emit: contigs
+    tuple val(meta), path("*.correctedReads.fasta.gz")	, emit: corrected_reads		, optional: true
+    tuple val(meta), path("*.trimmedReads.fasta.gz")	, emit: corrected_trimemd_reads	, optional: true
+    tuple val(meta), path("*.contigs.layout")    	, emit: metadata 		, optional: true
+    tuple val(meta), path("*.contigs.layout.readToTig")	, emit: contig_position 	, optional: true
+    tuple val(meta), path("*.contigs.layout.tigInfo")	, emit: contig_info		, optional: true
+    path "versions.yml"           			, emit: versions
```

Comment:
Usually that code is aligned nicely, somehow that looks a little chaotic here.
For me that isnt a bis problem though.